traffic_ops/app/lib/MojoPlugins/Daemonize.pm

```diff
@@ -41,12 +41,6 @@ sub register {
 				# This is the first child
 				$self->inactivity_timeout(0);
 				POSIX::setsid();
-				open( STDIN, "< /dev/null" )
-					|| confess("Can't read /dev/null: $!");
```

Comment:
I removed them because they were preventing me from seeing log output from the child process (even using $self->app->log).  It's probably just an issue with my local env, so I can add them back in.